src/components/editor/leftNav/ActivePanelPropertyNav.jsx

```diff
@@ -44,6 +45,20 @@ const ActivePanelPropertyNav = (props) => {
     return <ul>{subNavItems}</ul>
   }
 
+  const isModalOpen = useSelector((state) => selectModalType(state))
+
+  const handleClick = (property) => {
+    if (isModalOpen) return // do not respond to clicks in the nav if any modal is open
+
+    dispatch(
+      setCurrentComponent(
+        property.rootSubjectKey,
+        property.rootPropertyKey,
+        property.key
+      )
+    )
+  }
+
```

Comment:
This function is duplicated in other spots, would be better to refactor to be DRY